INSTRUCTIONS.native.md

```diff
@@ -0,0 +1,92 @@
+The following instructions are for setting up MSVC without docker.
+
+# FAQ
+
+## Does it run on Ubuntu 18.04 LTS?
+
+Yes, but the install scripts won't work because `msitools` is too old. You'll need to install either via Docker or on a real Ubuntu 20.04 LTS machine; and later copy paste the files under `/opt/msvc`
+
+## Does it work with CMake?
+
+Yes, but a custom version is needed or else CMake will complain that `/opt/msvc/bin/x64/cl` can't build a simple program.
+
+## Can it build Debug versions?
+
+Yes, but there may be troubles. From wine errors appearing in the logs/console to problems when trying to launch it because of missing debug redistributables.
+
+For the best experience build in Release mode
+
+## I get an "access denied" error when building RC via CMake.
+
+`Ninja` Generator has a problem when setting up RC. Use `Unix Makefile`s instead
+
+There is a [custom version of Ninja](https://github.com/mstorsjo/ninja/commits/cross-msvc) to deal with MSVC cross compiling but I haven't checked it to see if it fixes the problem.
```

Comment:
I'd kinda prefer to at least look into the issue before adding a FAQ entry saying "I haven't checked"...

(I haven't had time to look into your other comment on this yet - I'm a bit surprised that I haven't run into it, but maybe the cmake projects I've built haven't included any RC files?)

Reply:
I think we can remove this section about the RC issues now, by pointing the "custom cmake" at the newer commit that includes the cherrypicked fix for it.